OrderService/OrderService.Domain/Entities/ProductEntity.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace OrderService.Domain.Entities;
+
+public class ProductEntity
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+    public int Quantity { get; set; }
+    public double Price { get; set; }
+    public string? ImageUrl { get; set; }
+    public string PriceId { get; set; }
```

Comment:
What is the PriceId? Why do you need this field?

Reply:
This is id for Stripe paymen system, where i should register each product and get priceId (generated by stripe) from there to create payments